src/pages/overview/pdf-services-api/howtos/combine-pdf.md

```diff
@@ -219,6 +219,91 @@ const fs = require("fs");
 })();
 ```
 
+#### Python
+
+```python
+# Get the samples https://github.com/adobe/pdfservices-python-sdk-samples
+# Run the sample:
+# python src/combinepdf/combine_pdf.py
+
+import logging
```

Comment:
@jasnoors Please remove these imports in all operations. We just want to show the logic here.